<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aerobridge Database GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_config">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Point Selection</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>80</y>
        <width>691</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_open_ImgDialog">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Points</string>
      </property>
     </widget>
     <widget class="QPushButton" name="show_points">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Points</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_result">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Results</string>
     </attribute>
     <widget class="QTableView" name="display_table">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>681</width>
        <height>221</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="db_tables">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Eliptical Window</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rectangular Window</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Text</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Straight Arrow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Curved Arrow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Door Frame Lines</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Door Handle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Door Foorplate</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="detection_results">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>230</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>D:/Personal/NTU/Dissertation/Aerobridge/Database Images/Excel/CircularWindow/CircularWindowProcessed.csv</string>
      </property>
     </widget>
     <widget class="QPushButton" name="read_det_results">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>84</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_connect">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>84</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_database_path">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>230</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>C:/sqlite/aerobridge.db</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_dbpath">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Database Path:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_tablename">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Table</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_det_result_path">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Result File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_read_table">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>84</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Read Table</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_compare">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>100</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Compare</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>90</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>130</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>100</y>
        <width>20</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="show_point">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>100</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Points</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>390</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1.       Circular Window: 4 or 3 points. Ellipse w,h
2.       Rectangular Window. 4 points 
3.       Text : 4 points. 
4.       Straight Arrow : 3 points for triangle  and one point for rectangle
5.       Curved Arrow: 3 points for triangle  ,  3 points of the arc
6.       Door Frame Lines: 3 points, one at each end and one in the middle. 
7.       Door Handle:  6 points. Diamond shape
8.        Door Footplate: assume is rectangle.  4 points
</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
